.pre-commit-config.yaml

```
@@ -28,4 +29,4 @@ repos:
     rev: v1.3
     hooks:
     - id:  giticket
-      args: ['--regex=(\d+)',"--mode=regex_match", '--format={commit_msg} | [#{ticket}]']
+      args: ['--regex=(\d+)',"--mode=regex_match", '--format={commit_msg} | [#{ticket}]']
```

We shouldn't remov trailing lines. I think the `pre-commit run` should throw an error